manual.asc

```diff
@@ -80,14 +80,14 @@ Note that the currently selected deck is highlighted with a grey background, and
 === Other Deck Actions
 Long tapping on a deck will show a list of other actions available to perform on that deck:
 
-Rename deck :: Use this option to rename a deck
+Rename deck :: Use this option to rename a deck.
```

Comment:
Hi there! These line endings are not in agreement with each other. You used colons above and periods here. I propose using no line endings thoughtoug, as was the case before

Reply:
Alright, I will make that change!

Reply:
Alright, I will keep the changes on line number 45 and 56 and undo the rest of them!